core/orch_test.go

```diff
@@ -754,10 +754,15 @@ func defaultPaymentWithTickets(t *testing.T, senderParams []*net.TicketSenderPar
 	}
 
 	sender := pm.RandBytes(123)
+	expirationParams := &net.TicketExpirationParams{
```

Comment:
> I seem to have a preference to just define these within the TicketParams protobuf definition rather than creating an additional message that will never be passed over the wire by itself. Does this seem reasonable?

The `TicketParams` message contains fields that the recipient provides to the sender that would be fixed across all tickets in a payment. `creation_round` and `creation_round_block_hash` are set by the sender when creating tickets so I don't think the `TicketParams` message is the right place for those fields.

I defined an additional `TicketExpirationParams` message to group together `creation_round` and `creation_round_block_hash` since these fields come as a pair i.e. we cannot have one without the other for expiration validation. I thought the nested message better conveyed this relationship. Furthermore, a smaller additional benefit is that if we ever support multiple types of expiration params then we could wrap `TicketExpirationParams` in a `oneof` as mentioned in [this comment](https://github.com/livepeer/go-livepeer/pull/890#issuecomment-499135690).

> if we change the implementation for Creating Tickets around a bit to make optimal use of the new Payment construction

I agree that we should think about how to optimize the current ticket creation process which is now inefficient.

Reply:
>  I thought the nested message better conveyed this relationship.

Sounds reasonable. 

>  Furthermore, a smaller additional benefit is that if we ever support multiple types of expiration params then we could wrap TicketExpirationParams in a oneof

Since expiration is so tightly coupled to the protocol implementation I don't see this happening any time soon and the only  alternative I can think of is future block timestamps. Then again more flexibilty from the get-go sounds good. 

Thanks for claryfing ! approved.